sql/core/src/main/java/org/apache/spark/sql/execution/vectorized/AggregateHashMap.java

```diff
@@ -19,6 +19,8 @@
 
 import java.util.Arrays;
 
+import com.google.common.annotations.VisibleForTesting;
```

Comment:
just to annotate `public int find(long key)` that is only used in `BenchmarkWholeStageCodegen`.